api/proto/common/common.proto

```diff
@@ -29,13 +29,20 @@ message BlobCommitment {
 	uint32 data_length = 4;
 }
 
+message BlobHeader {
+	// Blob version
+	uint32 version = 1;
+	repeated uint32 quorum_numbers = 2;
+	common.BlobCommitment commitment = 3;
+	bytes payment_header_hash = 4;
+}
+
 // BlobCertificate is what gets attested by the network
 message BlobCertificate {
-	uint32 version = 1;
-	bytes blob_key = 2;
-	BlobCommitment blob_commitment = 3;
-	repeated uint32 quorum_numbers = 4;
+	bytes blob_key = 1;
```

Comment:
Is the blob key going to be equivalent to the blob hash? If "yes", perhaps if we should just call this field `blob_hash`.

Reply:
`blob_key` is used throughout the system as a `hash(blob_header)`. What's the blob hash?

Reply:
Makes sense, I think I was mistaken about the blob hash thing.